openwisp_monitoring/monitoring/models.py

```diff
@@ -500,9 +486,9 @@ def read(
             else:
                 data_query = self.get_query(**query_kwargs)
                 summary_query = self.get_query(summary=True, **query_kwargs)
-            points = list(query(data_query, epoch='s').get_points())
-            summary = list(query(summary_query, epoch='s').get_points())
-        except InfluxDBClientError as e:
+            points = list(TimeseriesDB.query(data_query, epoch='s').get_points())
+            summary = list(TimeseriesDB.query(summary_query, epoch='s').get_points())
+        except TIMESERIES_DB_EXCEPTION as e:
```

Comment:
Please use CamelCase for the exception name and let's keep it less generic. What about TimeseriesDbClientError?